src/controllers/auth-controller.ts

```diff
@@ -202,6 +207,61 @@ export default class AuthController extends AbstractController {
     }
   }
 
+  @POST
+  @Path('/forgot')
```

Comment:
in this controller, I also updated the methods when I merged the master. But there is also a need to declare more strictly, create the necessary types and check as it is understood by the swagger.